views/js/controller/PrintTest/render.js

```diff
@@ -102,6 +103,31 @@ define([
         });
     }
 
+    /**
+     * Extract shared stimulus from a item, put it in section before a item
+     * Remove repeated shared stimulus
+     * @param {JQuery} $container
+     */
+    function extractSharedStimulus($container) {
+        const hrefs = {};
+        // find shared stimulus
+        $container.find('.qti-include').each(function () {
+            const $include = $(this);
+            const href = $include.attr('data-href');
+            if (!hrefs[href]) {
+                hrefs[href] = true;
+                const $section = $include.parents('section');
```

Comment:
I believe we should use `closest` method. 
Could be unexpected behavior in case of nesting `section` elements

Reply:
fixed